README.md

```diff
@@ -59,6 +59,62 @@ Do the following to initialize a project:
 1. Search for flutter_template and replace it with the name of your project where needed.
 2. Initialize FirebaseCrashlytics (you can find that in TODO(init-project)).
 
+#### Firebase
+
+To use Firebase in your project, you need to do the following:
+
+1. If you haven't already, install [Firebase CLI](https://firebase.google.com/docs/cli).
+2. Log in to Firebase CLI with your Google account:
+   ```sh
+   firebase login
+   ```
+3. If you haven't already, activate the FlutterFire CLI plugin:
+   ```sh
+   dart pub global activate flutterfire_cli
+   ```
+4. Create project at [Firebase Console](https://console.firebase.google.com/).
+5. Execute command to create a new project in Firebase or select existing and specify supported platforms:
+   ```sh
+   flutterfire configure -p name_or_id_of_your_project --platforms android,ios -i your.ios.bundleId -a your.android.package.name
```

Comment:
We always have different flavors in app, is this will be used?

Reply:
below there is a command for config generation for certain flavor:
    ```sh
    flutterfire configure -p name_or_id_of_your_project -o lib/firebase_options_dev.dart --platforms android,ios -i your.ios.bundleId.dev -a your.android.package.name.dev
    ```

Reply:
fix this